src/couchdb/couch_httpd_db.erl

```diff
@@ -1015,8 +1015,8 @@ db_attachment_req(#httpd{method=Method,mochi_req=MochiReq}=Req, Db, DocId, FileN
     _ ->
         [{"Location", absolute_uri(Req, "/" ++
             ?b2l(Db#db.name) ++ "/" ++
```

Comment:
Yes, but there is an `/` character this need to be escaped. See my comment in PR discussion.